bayer_method: ''
bayer_pattern: rggb
frame_id: /camera
frame_rate: 60.0
format7_color_coding: raw8
gain: 12.041193008422852
gamma: 1.2000000000000002
guid: 00b09d01008b7769
use_ros_time: true
video_mode: format7_mode0

camera:
    focal_length: 4.00
    image_width: 640
    image_height: 480
    detPitch: 6.0

processingScale: 2
tgtEdge: 0.05
circleRadius: 0.02
lambda:
    x: 0.02
    y: 0.02
    z: 0.02
    r: 0.02
    p: 0.02
    yaw: 0.02
zOffset: 0.5
majorToMinor: 1.50